Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -525,6 +525,9 @@ open class CarPlayNavigationViewController: UIViewController, BuildingHighlighti
         if let coordinate = navigationService.routeProgress.route.shape?.coordinates.first {
             navigationMapView.setInitialCamera(coordinate)
         }
+        
+        // TODO: remove when implement alternatives selection on CarPlay
```

Comment:
The way this is typically done on CarPlay is via a CPListTemplate listing the other routes for the user to select. Even then, it would probably be reasonable to display the alternative routes non-interactively on the map.